src/dex-ui/services/HederaService/HederaService.ts

```diff
@@ -76,139 +92,142 @@ function createHederaService() {
       addLiquidityContractAddress,
       walletAddress,
       signer,
-    } = addLiquidityDetails
-      ? addLiquidityDetails
-      : {
-          firstTokenAddress: null,
-          firstTokenQuantity: null,
-          secondTokenAddress: null,
-          secondTokenQuantity: null,
-          addLiquidityContractAddress: null,
-          walletAddress: null,
-          signer: null,
-        };
+    } = addLiquidityDetails;
 
     // TODO: remove fallbacks if no signer and not all details are provided
     const firstTokenAddr = firstTokenAddress ? firstTokenAddress : tokenA;
     const secondTokenAddr = secondTokenAddress ? secondTokenAddress : tokenB;
     const firstTokenQty = firstTokenQuantity ? firstTokenQuantity : new BigNumber(10);
     const secondTokenQty = secondTokenQuantity ? secondTokenQuantity : new BigNumber(10);
     const addLiquidityContractId = addLiquidityContractAddress ? addLiquidityContractAddress : contractId;
-    const fromAddress = walletAddress ? walletAddress : treasure;
+    const fromAddress = walletAddress ? walletAddress : treasuryKey;
```

Comment:
dont think we need this anymore if we are removing the fallback from the signer not being there - always gonna go with the signer and not the client so fromAddress and treasuryKey isn't being used

Reply:
Yea, removed.